localisation/00_zuwc_united_localisation.csv

```diff
@@ -2645,14 +2645,42 @@ GetInSociety_society; in [This.Society.GetName];;;;;;;;;;;;;x
 GetInSocietyYellow_society; in �Y[This.Society.GetName]�!;;;;;;;;;;;;;x
 GetSocietyMembers_society;Society members;;;;;;;;;;;;;x
 GetSocietyMembers_brethren;brethren;;Br�der;;;;;;;;;;;x
-GetSocietyWe_society;[This.Society.GetName];;;;;;;;;;;;;x
-GetSocietyWe_we;we;;wir;;;;;;;;;;;x
-wc_united_sacrifice_all_prisoners_decision;Perform a Mass Sacrifice;;Ein Massenopfer darbringen;;;;;;;;;;;x
-wc_united_sacrifice_all_prisoners_decision_desc;Sacrifice all your prisoners to [Root.Religion.GetHighGodName].;;Opfert all Eure Gefangenen [Root.Religion.GetHighGodName].;;;;;;;;;;;x
-GetHighwaymenType_dragon_hunters;dragon hunters;;Drachenj�ger;;;;;;;;;;;x
-GetHighwaymenType_monster_hunters;monster hunters;;Monsterj�ger;;;;;;;;;;;x
-GetHighwaymenType_witch_hunters;witch hunters;;Hexenj�ger;;;;;;;;;;;x
-GetHighwaymenType_highwaymen;highwaymen;;Vagabunden;;;;;;;;;;;x
+GetSocietyWe_we;we;;;;;;;;;;;;;x
+EVTTITLE_WCVRK_110;Till Death Do Us Part;;Bis das der Tod uns scheidet;;;;;;;;;;;x
+EVTDESC_WCVRK_110;Recently, the illness that has befallen your husband has been growing worse with each passing day. Knowing well that he might not last another week, you have sought many solutions, be it magical or alchemical, but nothing seems to help alleviate his condition. The grief over your inevitable loss overcomes you, and in desperation you cry out in hopes something or someone will hear your pleas. To your surprise, a shadowy figure emerges and makes you a promise that death will never  part you and your husband... for a price. He offers you an unimaginable power and eternal life in exchange for loyalty, and you realize this might be the only way to keep your husband at your side. Conflicting thoughts cross your mind, but ultimately realize that if death can part you both...;;;;;;;;;;;;;x
+EVTOPTA_WCVRK_110;...then it might as well keep you together, too.;;;;;;;;;;;;;x
+EVTOPTB_WCVRK_110;...then accept the fact and refuse the creature's offer.;;;;;;;;;;;;;x
+EVTTITLE_WCVRK_111;Till Death Do Us Part;;Bis das der Tod uns scheidet;;;;;;;;;;;x
+EVTDESC_WCVRK_111;Your illness has been progressing in a steady pace and you've already been coming to terms that you might not survive. However, one day you notice your wife entering your chambers and immediately recognize that she's unusually cheerful. Suddenly, you see a horrific spectre looming above her. 'Don't be startled, love. I have found a way to cure your illness. You no longer have to worry about leaving me... We'll be together forever.' You attempt to flee but you notice that the dark magic is binding you. You hear your wife chanting something in an unrecognizable language and after a brief moment of darkness, you come back to your senses, although you notice something has changed. You are dead, but at the same time you are not. As you struggle to understand your situation, you notice that your wife has been also changed into an apparition.;;;;;;;;;;;;;x
```

Comment:
This happens only for sich husband?

Reply:
WCVRK_110 is event for wife, WCVRK_111 is event for husband. Both are intended recipients. The husband needs to fall ill, then the wife makes a deal with Gorak's spirit, as it was in lore.

Reply:
Why can't husband get the deal and become deathshaper like Gorak?

Reply:
There is a catch in the deal. In lore, Arthur Waycrest was raised as a mindless husk. Husband accepting the offer would make the point of Heartsbane Coven null, since there is no capable 'Mother' to lead the coven. The only solution is changing the detrimental result of the deal.